openshift-install-powervs

```diff
@@ -1422,6 +1477,12 @@ function main {
       warn "Enabling forceful destruction option for terraform destroy command"
       FORCE_DESTROY=1
       ;;
+    "-flavor")
+      shift
+      flavortype="$1"
+      vars+=" -var-file ../$varfile -var-file ../automation/compute-vars/$flavortype.tfvars"
```

Comment:
What if I give `-flavor` option before `-varfile` ? I feel `varfile` will be "".